test/metric/processes.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package metric
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+)
+
+type ProcessesMetricValueFetcher struct {
```

Comment:
I thought processes was already merged.. does this need to be rebased?

Reply:
Not sure why its show me up here to. I have already rebased and merged though hmmm.

Reply:
Its weird that I have rebased and the diff PR commits does reflect it but now here https://github.com/aws/amazon-cloudwatch-agent-test/compare/main...khanhntd:amazon-cloudwatch-agent-test:collectd_sanity